data/fonts/README.txt

```diff
@@ -0,0 +1,364 @@
+----------------------------------------------------------------------
```

Comment:
I'd remove everything from this directory except for the `ProggyClean.ttf`. Unnecessary and irrelevant now that the font file is hardcoded anyway. Tho it might be good to credit 

> `ProggyClean.ttf` by Tristan Grimmer

somewhere, although I don't know if there's even such a place in the docs or README.